docker/Dockerfile

```diff
@@ -19,3 +15,13 @@ RUN chmod +x /wait
 
 RUN pip install --no-cache-dir -r requirements/dev.txt \
     && rm -rf /tmp/*
+
+COPY . .
+
+EXPOSE 5000
```

Comment:
always declare ports with `EXPOSE`, unambiguous what the container listens on :+1: